README.md

```diff
@@ -49,18 +48,26 @@ pp count = types.size # => 5
 session.stop
 ```
 
+## Testing Selenium Webdriver Crystal
+
+- Install Chromedriver
```

Comment:
We should probably include a link to the chromedriver homepage: http://chromedriver.chromium.org/home